File: Data Structures & Algorithms/concatenation-of-array/submission-1.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int n = nums.length;
        int[] ret = new int[2*n];
        for( int x = 0 ; x < n ; x++ ){
            ret[x] = nums[x];
            ret[x+n] = nums[x];
        }
        return ret;
    }
}

File: Data Structures & Algorithms/two-integer-sum/submission-1.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        dict = {}
        index = {}
        ret = []

        for i, x in enumerate(nums):
            dict[x] = target - x
            index[x] = i


        for i, x in enumerate(nums):
            temp = target - x
            if temp in index and index[temp] != i:
                ret.append(i)
                ret.append(index[temp])
                return sorted(ret)